pkg/distributor/distributor.go

```diff
@@ -351,7 +329,46 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 		return d.ingestionRate.Rate()
 	})
 
-	d.forwarder = forwarding.NewForwarder(reg, d.cfg.Forwarding)
+	// Create the configured ingestion rate limit strategy (local or global). In case
+	// it's an internal dependency and can't join the distributors ring, we skip rate
```

Comment:
:question: ... and WE can't ...

Reply:
Oops. Must have gotten lost during a refactor.